tasks/main.yml

```diff
@@ -22,16 +30,31 @@
     mode: 0440
   sudo: yes
   sudo_user: "{{ server_user }}"
+  tags: test
 
-- unarchive:
+- name: Decompress apache maven
+  unarchive:
     src: /tmp/apache-maven.tar.gz
-    dest: /home/tinkerware/
+    dest: /opt/
     copy: no
+    owner: "{{ server_user }}"
+    group: "{{ server_user }}"
+    mode: 0755
+  sudo: yes
+  sudo_user: root
+
+- name: Add apache maven path
+  shell: touch /etc/profile.d/spring.sh && echo "if [ -d \"/opt/apache-maven-3.5.0/bin\" ]; then" > /etc/profile.d/spring.sh  && echo "  PATH=\"$PATH:/opt/apache-maven-3.5.0/bin/\"" >> /etc/profile.d/spring.sh && echo "fi" >> /etc/profile.d/spring.sh
+  sudo: yes
+
+- name: Download sample project
+  git:
+    repo: "https://github.com/spring-projects/spring-petclinic.git"
+    dest: "/home/{{ server_user }}/samples/spring-petclinic"
```

Comment:
this should be a variable as well. In case of the project is a different one.
check into filters. `{{ git_repo | basename }}`
Something like that should work